src/IoTHub.Portal.Infrastructure/Jobs/SendPlanningCommandJob.cs

```diff
@@ -135,7 +135,7 @@ public void AddNewDevice(DeviceListItem device)
         {
             var layer = layers.FirstOrDefault(layer => layer.Id == device.LayerId);
 
-            if (layer?.Planning != "None")
+            if (layer?.Planning is not "None" and not null)
```

Comment:
I would prefer to invert conditions 
```suggestion
            if (layer?.Planning is not null and not "None")
```